app/controllers/admin/appointments_controller.rb

```diff
@@ -8,15 +8,16 @@ def index
       @appointments = Appointment
         .where(starts_at: @current_day.beginning_of_day..@current_day.end_of_day)
         .chronologically
-        .includes(:member, loans: {item: :borrow_policy}, holds: {item: :borrow_policy})
+        .includes(:member, :same_day_and_member, loans: {item: :borrow_policy}, holds: {item: :borrow_policy})
 
       @pending_appointments = []
       @completed_appointments = []
-      sort_by_member_and_time(@appointments).each_with_index do |appointment, index|
+
+      @appointments.sort_by { |a| helpers.appointment_sort_key(a) }.each do |appointment|
```

Comment:
This is a little funky, but I figured it made sense to use the same sorting technique here that we do on the front end.